wallet/udb/common_test.go

```diff
@@ -44,32 +47,43 @@ func hexToBytes(origHex string) []byte {
 }
 
 // createEmptyDB is a helper function for creating an empty wallet db.
-func createEmptyDB() error {
-	_ = os.Remove(emptyDb)
+func createEmptyDB() (string, error) {
+	var err error
 
-	db, err := walletdb.Create("bdb", emptyDb)
-	if err != nil {
-		return fmt.Errorf("unexpected error: %v", err)
-	}
+	doOnce.Do(func(){
```

Comment:
if createEmptyDB is only called once before tests run, it doesn't need sync.Once.  The point of sync.Once is that it could create a db in a process-specific temp path lazily, and that the entire directory could be removed (if it was ever created) by TestMain